integration_tests/steps/device.py

```diff
@@ -85,17 +89,20 @@ def step_impl(context):
     # placing bids and offers on every market cycle.
     # Should stop if an error occurs or if the simulation has finished
     counter = 0  # Wait for five minutes at most
-    while context.aggregator.errors == 0 and context.aggregator.status != "finished" and counter < 60:
+    while (
+            len(context.aggregator.errors) == 0
```

Comment:
Now we count the number of errors in the list rather than just checking the integer.